mushr_utils/install/mushr_install.bash

```diff
@@ -102,15 +119,21 @@ cd $MUSHR_WS_PATH/catkin_ws/src/ && vcs import < mushr/base-repos.yaml && vcs im
 
 # Make custom mushr_noetic script
 cat <<- EOF > ${MUSHR_INSTALL_PATH}/mushr_noetic
+#!/bin/bash
 export MUSHR_INSTALL_PATH=${MUSHR_INSTALL_PATH}
 export MUSHR_REAL_ROBOT=${MUSHR_REAL_ROBOT}
 export MUSHR_WS_PATH=${MUSHR_WS_PATH}
 export MUSHR_COMPOSE_FILE=${MUSHR_COMPOSE_FILE}
 export MUSHR_OS_TYPE=${MUSHR_OS_TYPE}
-if [ \$# == 0 ] || [ \$1 == "run" ];
+if [[ \$# == 0 ]] || [[ \$1 == "run" ]];
 then
-  docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash
-elif [ \$1 == "build" ];
+  if [[ \$# == 2 ]];
+  then
+    docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash -ic "\${2}"
+  else
+    docker-compose -f \$MUSHR_INSTALL_PATH/\$MUSHR_COMPOSE_FILE run -p 	9090:9090 mushr_noetic bash
+  fi
+elif [[ \$1 == "build" ]];
```

Comment:
do we need this additional elif? could we not put it in the prior one?

Reply:
What do you mean bt putting it in the prior one? How did you imagine handling `mushr_noetic build`?

I tried to leave the existing logic as intact as possible, and just added one more case that can handle `mushr_noetic run {custom_bash_command}` so I can execute bash commands within the container brought up by `mushr_noetic` noninteractively.

Essentially all the changes I made to this script is to support "noninteractive" mode for `mushr_noetic` so it can be used programmatically